apps/backend/src/utils/middleware.ts

```diff
@@ -1,40 +1,49 @@
-import { Request, Response, NextFunction } from 'express';
+import { Request, Response, NextFunction } from "express";
 
-const logger = require('./logger')
+import logger from "./logger";
 
-export const requestLogger = (request: Request, response: Response, next: NextFunction) => {
-    logger.info('Method:', request.method)
-    logger.info('Path:  ', request.path)
-    logger.info('Body:  ', request.body)
-    logger.info('---')
-    next()
-}
+export const requestLogger = (
+  request: Request,
+  response: Response,
+  next: NextFunction
+) => {
+  logger.info("Method:", request.method);
+  logger.info("Path:  ", request.path);
+  logger.info("Body:  ", request.body);
+  logger.info("---");
+  next();
+};
 
 // Shows a 404 error for a note with unknown id
 export const unknownEndpoint = (request: Request, response: Response) => {
-    response.status(404).send({ error: 'unknown endpoint' });
+  response.status(404).send({ error: "unknown endpoint" });
 };
 
-export const errorHandler = (error: any, request: Request, response: Response, next: NextFunction) => {
-    if (error.name === 'CastError') {
-        return response.status(400).send({
-            error: 'malformatted id',
-        });
-    } else if (error.name === 'ValidationError') {
-        return response.status(400).json({
-            error: error.message,
-        });
-    } else if (error.name === 'JsonWebTokenError') {
-        return response.status(401).json({
-            error: 'invalid token',
-        });
-    } else if (error.name === 'TokenExpiredError') {
-        return response.status(401).json({
-            error: 'token expired',
-        });
-    }
+export const errorHandler = (
+  error: any,
+  request: Request,
+  response: Response,
+  next: NextFunction
+) => {
+  if (error.name === "CastError") {
+    return response.status(400).send({
+      error: "malformatted id",
+    });
+  } else if (error.name === "ValidationError") {
+    return response.status(400).json({
+      error: error.message,
+    });
+  } else if (error.name === "JsonWebTokenError") {
+    return response.status(401).json({
+      error: "invalid token",
+    });
+  } else if (error.name === "TokenExpiredError") {
+    return response.status(401).json({
+      error: "token expired",
```

Comment:
```suggestion
  if (error.name === ServerError.CAST_ERROR) {
    return response.status(400).send({
      error: "malformatted id",
    });
  } else if (error.name === ServerError.VALIDATION_ERROR) {
    return response.status(400).json({
      error: error.message,
    });
  } else if (error.name === ServerError.VALIDATION_ERROR) {
    return response.status(401).json({
      error: "invalid token",
    });
  } else if (error.name === ServerError.TOKEN_EXPIRED_ERROR) {
    return response.status(401).json({
      error: "token expired",
```

Try to avoid the use of magic strings. So putting them as a enum to import might be a better idea.
e.g.
```
enum ServerError {
  CAST_ERROR = "CastError"
...
}
```

Reply:
Good catch, danke schön